app/controllers/api/v2/compliance/arf_reports_controller.rb

```diff
@@ -31,7 +31,7 @@ def get_resource
         end
 
         resource_description do
-          resource_id 'foreman_openscap_arf_reports'
+          resource_id 'arf_reports'
```

Comment:
No, tbh.
I believe that all of the resource_description can be moved (from all of the controllers)